src/main/java/ie/cit/teambravo/cardsec/services/ValidationServiceImpl.java

```diff
@@ -27,7 +34,8 @@ public boolean validate(String panelId, String cardId, Boolean allowed) {
 		eventDto.setAccessAllowed(allowed);
 		eventDto.setTimestamp(System.currentTimeMillis());
 		eventDto.setLocation(panelLocatorService.getPanelLocation(panelId));
-
+		durationService.getTravelTimeBetween2Points(new LatLngAlt(36.12, -86.67, 10.0),
```

Comment:
sth that I was thinking about: would it be worth to label these magic numbers? aka assign to a nice variable with a nice name?

Reply:
Oh they are just there so something happens for now when you call the API. Can drop them once we get data down from the controller.